invenio_app_ils/circulation/api.py

```diff
@@ -104,6 +104,9 @@ def request_loan(
     **kwargs
 ):
     """Create a new loan and trigger the first transition to PENDING."""
+    if not current_user.is_authenticated:
```

Comment:
Very very very minor, for the future, I generally tend to prefer "positive" assertion, so I would have changed this to:
```suggestion
    if current_user.is_anonymous:
```